runelite-client/src/main/java/net/runelite/client/plugins/fishing/FishingPlugin.java

```diff
@@ -83,6 +85,7 @@ public class FishingPlugin extends Plugin
 	private static final int TRAWLER_SHIP_REGION_NORMAL = 7499;
 	private static final int TRAWLER_SHIP_REGION_SINKING = 8011;
 	private static final int TRAWLER_TIME_LIMIT_IN_SECONDS = 314;
+	private static final Pattern CAUGHT_KARAMBWANJI_MESSAGE = Pattern.compile("You catch (\\d+) Karambwanji.");
```

Comment:
Let's adapt this to match all standard catch messages.
```suggestion
	private static final Pattern CAUGHT_FISH_MESSAGE = Pattern.compile("You catch (?:a|some|\\d+ Karambwanji\\.)");
```

This change lets us simplify the condition where it's used to:

```java
if (message.equals("Your cormorant returns with its catch.") || m.find())
```